src/definitions/modules/dimmer.less

```diff
@@ -302,7 +302,6 @@ body.dimmable > .dimmer {
     .ui[class*="center dimmer"] {
         top: 50%;
         transform: translateY(-50%);
-        -webkit-transform: translateY(calc(-50% - 0.5px));
```

Comment:
https://github.com/fomantic/Fomantic-UI/pull/186/commits/678695ed447e630a70934ee1b3df5c4fe91a2109

is this hack still needed?

Reply:
Doesnt seem so after a quick test